hs/CssGen.hs

```diff
@@ -7,14 +7,18 @@ import Prelude hiding ((**))
 import Data.Monoid
 import Data.Text.Lazy
 import System.Directory
-
+import Consts
+import CommonCssGen as COMMON
```

Comment:
Not big on the all-caps here. Just import the one style per module, e.g.

`import CommonCssGen (common)`